scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -182,6 +182,10 @@
           SPIDEX
           <span class="float-right">{{ variant.spidex_human }}</span>
         </li>
+        <li class="list-group-item">
+          SpliceAI DS Max
+          <span class="float-right">{{ variant.spliceai_ds_max }}</span>
```

Comment:
We're saving the variant spliceai string (bla|bla|bla|..) but we're not showing it on the variant page. Wouldn't users be interested in those values?

Reply:
Do you know what values in particular? But, presumably I would be interested in all. Perhaps a hover? I'll try something!

Reply:
A hover would be perfect!

Reply:
Hm, I wonder if they will get that? I would probably have to look up the VCF-info key and then read the SpliceAI-paper to understand, but at least now it is displayed in some form.